codegen/rate_limiter.go

```diff
@@ -0,0 +1,45 @@
+package codegen
+
+import (
+	"github.com/solo-io/skv2/codegen/model"
+	"github.com/solo-io/skv2/contrib"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func init() {
+	groups = append(groups, rateLimiterGroups()...)
+}
+
+func rateLimiterGroups() []model.Group {
+	return []model.Group{
+		{
+			GroupVersion: schema.GroupVersion{
+				Group:   "ratelimit.solo.io",
+				Version: "v1alpha1",
+			},
+			Module: "github.com/solo-io/solo-apis", // TODO
```

Comment:
why the TODO?

Reply:
Forgot to remove it, it was a personal note to check how the field is used.